src/main/java/competition/subsystems/arm/ArmPositionState.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ArmPositionState {
```

Comment:
as we talked about in person, we probably want a way for this state to return that the requested position is impossible (rather than crashing). Maybe an enum here? Agree it could come later